lite-pm/styles/TeamMember.module.css

```diff
@@ -0,0 +1,26 @@
+.box {
+  border-width: 2px;
+  border-style: solid;
+  border-color: rgb(192, 185, 185);
+  border-radius: 15pt;
+  width: 110pt;
+  float: left;
```

Comment:
No need to change, but remember in the future that most of these CSS styles can be `classNames` from `bootstrap`. E.g. instead of setting `margin-left: 8pt;` here you can do `className="ml-2"`

Reply:
className={styles.someStyle}

Reply:
I think `className={[styles.someStyle, "ml-2"]}` would work